### PR TITLE
Merge `next`

### LINKED ISSUE #105: Publish @next as @latest
I need help! Could people please try upgrading to `p-s@next` and tell us about your experience? I'll come back to this issue when I have a second to add a migration guide (I'll actually put it in docs somewhere).

Thanks!

```
npm install p-s@next
```

### PR DESCRIPTION
**Pull request by kentcdodds**
Closes #105 

Unfortunately I think this may go out as `v5` because I accidentally published the beta as `v4`, but in any case, this is the next major version with a few changes. I've been using it exclusively for a little while and I think we're good. What still needs to happen to merge (and auto-release) this?

### CODE REVIEW

**Comment by kentcdodds** [comment]
I think that once this is merged, I'll edit the GitHub release with a migration guide. Should just be a few notes, but it'll make moving over easier.

File: cli-test/__snapshots__/cli.test.js.snap [hunk 1]
```diff
@@ -40,19 +40,18 @@ Object {
...
+    -c, --config <filepath>  Config file to use (defaults to nearest package-scripts.yml or package-scripts.js)
+    -l, --log-level <level>  The log level to use (error, warn, info [default])
+    -r, --require <module>   Module to preload
 
 Available scripts (camel or kebab case accepted)
 
-test - echo \"test script\"
-lint - echo \"lint.default\"
-lint.sub.thing - this is a description - echo \"deeply nested thing\"
+[32mtest[39m - [90mecho \"test script\"[39m
```
Comment (addityasingh) [inline cli-test/__snapshots__/cli.test.js.snap]: Are the timing numbers expected or typo?

Comment (kentcdodds) [inline cli-test/__snapshots__/cli.test.js.snap]: Actually, those are special characters for coloring the output. That's from `chalk`. The editor won't show those characters. 😀

Comment (addityasingh) [inline cli-test/__snapshots__/cli.test.js.snap]: Okk. I was wondering if they got there unintentionally. Cool 👍

**Comment by addityasingh** [comment]
LGTM

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "p-s",
+  "name": "nps",
```
Comment (kentcdodds) [inline package.json]: It's official! I now have publish access to `nps`. I'll be renaming this package and we'll all be a lot less confused! Added bonus, I can start at v1.0.0 too!

File: .all-contributorsrc [hunk 3]
```diff
@@ -228,6 +228,15 @@
       "contributions": [
         "bug"
       ]
+    },
+    {
+      "login": "addityasingh",
```
Comment (kentcdodds) [inline .all-contributorsrc]: Thanks for the review @addityasingh

**Comment by kentcdodds** [comment]
Whoa... Ehhh... Something weird happened and it looks like this got published!? Haha. Whelp! I wanted to publish it sooner or later anyway, may as well be now. Alrighty, merging, and renaming the repo! Boy, I just signed myself up for a bit of work before I was ready for it!

**Pull request merged**